[Bug]: Fade example missing dependency and missing vanilla examples

### Which variants of Embla Carousel are you using?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [X] embla-carousel-docs **(Generator)**

### Steps to reproduce

Click **edit source** then select the React CodeSandbox and you should see it fails to run due to missing `embla-carousel-fade` dependency. This is probably is quick fix and I would be willing to create a PR if you point me in the right direction.
Also, I think the Vanilla examples are missing.

### Expected Behavior

Fancy feature should have...

### Additional Context

I have also tried...

### What browsers are you seeing the problem on?

_No response_

### Version

_No response_

### CodeSandbox

_No response_

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for bug reports

I think this is true outside of examples. I've tried to install the fade plugin with the latest version of embla and get the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: embla-carousel@8.1.0
npm ERR! node_modules/embla-carousel
npm ERR!   peer embla-carousel@"8.1.0" from embla-carousel-auto-scroll@8.1.0
npm ERR!   node_modules/embla-carousel-auto-scroll
npm ERR!     embla-carousel-auto-scroll@"^8.0.4" from the root project
npm ERR!   peer embla-carousel@"8.1.0" from embla-carousel-class-names@8.1.0
npm ERR!   node_modules/embla-carousel-class-names
npm ERR!     embla-carousel-class-names@"^8.0.4" from the root project
npm ERR!   2 more (embla-carousel-reactive-utils, embla-carousel-vue)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! embla-carousel-fade@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: embla-carousel@8.0.2
npm ERR! node_modules/embla-carousel
npm ERR!   peer embla-carousel@"8.0.2" from embla-carousel-fade@8.1.0
npm ERR!   node_modules/embla-carousel-fade
npm ERR!     embla-carousel-fade@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/packages/embla-carousel-docs/src/components/Sandbox/Vanilla/SandboxFilesSrc/Fade/EmblaCarousel.ts
+++ b/packages/embla-carousel-docs/src/components/Sandbox/Vanilla/SandboxFilesSrc/Fade/EmblaCarousel.ts
@@ -1,0 +1,31 @@
+import EmblaCarousel, { EmblaOptionsType } from 'embla-carousel'
+import Fade from 'embla-carousel-fade'
+import { addDotBtnsAndClickHandlers } from '../EmblaCarouselDotButton'
+import { addPrevNextBtnsClickHandlers } from '../EmblaCarouselArrowButtons'
+import '../css/base.css'
+import '../css/sandbox.css'
+import '../css/embla.css'
+
+const OPTIONS: EmblaOptionsType = {}
+
+const emblaNode = <HTMLElement>document.querySelector('.embla')
+const viewportNode = <HTMLElement>emblaNode.querySelector('.embla__viewport')
+const prevBtn = <HTMLElement>emblaNode.querySelector('.embla__button--prev')
+const nextBtn = <HTMLElement>emblaNode.querySelector('.embla__button--next')
+const dotsNode = <HTMLElement>document.querySelector('.embla__dots')
+
+const emblaApi = EmblaCarousel(viewportNode, OPTIONS, [Fade()])
+
+const removePrevNextBtnsClickHandlers = addPrevNextBtnsClickHandlers(
+  emblaApi,
+  prevBtn,
+  nextBtn
+)
+const removeDotBtnsAndClickHandlers = addDotBtnsAndClickHandlers(
+  emblaApi,
+  dotsNode
+)
+
+emblaApi
+  .on('destroy', removePrevNextBtnsClickHandlers)
+  .on('destroy', removeDotBtnsAndClickHandlers)

--- a/packages/embla-carousel-docs/src/consts/sandbox.ts
+++ b/packages/embla-carousel-docs/src/consts/sandbox.ts
@@ -113,7 +113,8 @@ export const SANDBOX_PLUGINS = {
   AUTOPLAY: 'embla-carousel-autoplay',
   AUTO_SCROLL: 'embla-carousel-auto-scroll',
   AUTO_HEIGHT: 'embla-carousel-auto-height',
-  CLASS_NAMES: 'embla-carousel-class-names'
+  CLASS_NAMES: 'embla-carousel-class-names',
+  FADE: 'embla-carousel-fade'
 } as const
 
 export const SANDBOX_GENERATOR_FORM_FIELDS = {

